projects/pastanaga-angular/src/lib/button/button.component.spec.ts

```diff
@@ -24,9 +24,7 @@ describe('ButtonComponent', () => {
         expect(component._iconSize).toEqual(Size.large);
         component.size = Size.medium;
         expect(component._iconSize).toEqual(Size.medium);
-    });
-    it('should set the icon size to medium for small buttons', () => {
         component.size = Size.small;
-        expect(component._iconSize).toEqual(Size.medium);
+        expect(component._iconSize).toEqual(Size.small);
```

Comment:
@mpellerin42 do you know if we had any specific reason why we assigned the medium size icon to the small size button?

Reply:
it was designed that way on the first place by Glow.